console/console-server/src/main/resources/iot.schema.graphql

```diff
@@ -0,0 +1,222 @@
+enum ProjectPhaseType {
+  Active
+  Configuring
+  Terminating
+  Failed
+}
+
+enum IotProjectType {
+  managed
+  external
+}
+
+enum IotCredentials {
+  psk
+  hashed_password
+  x509
+}
+
+enum IotEndpointName {
+  HttpAdapter
+  MqttAdapter
+  AmqpAdapter
+  CoapAdapter
+  DeviceRegistrationManagement
+  DeviceCredentialManagement
+}
+
+union IotProjectDownStreamStrategy =
+  ExternalDownstreamStrategy_iot_enmasse_io_v1alpha1 | ManagedDownstreamStrategy_iot_enmasse_io_v1alpha1
+
+type Credentials_iot_enmasse_io_v1alpha1{
+  username: String!
+  password: String!
+}
+
+type ConnectionInformation_iot_enmasse_io_v1alpha1{
+  host: String!
+  port: Int!
+  credentials: Credentials_iot_enmasse_io_v1alpha1!
+
+  tls : Boolean!
+  certificate: String
+}
+
+type ManagedStatus_iot_enmasse_io_v1alpha1 {
+  passwordTime: String #Date
+  addressSpace: String
+}
+
+type IoTProjectStatus_iot_enmasse_io_v1alpha1 {
+  phase: ProjectPhaseType!
+  phaseReason: String
+  tenantName: String!
+  downstreamEndpoint: ConnectionInformation_iot_enmasse_io_v1alpha1!
+
+  managedStatus: ManagedStatus_iot_enmasse_io_v1alpha1
+}
+
+type AddressSpaceConfig_iot_enmasse_io_v1alpha1  {
+  name: String!
+  plan: String!
+  type: String!
+}
+
+type AddressConfig_iot_enmasse_io_v1alpha1  {
+  plan: String!
```

Comment:
it seems there should a field for name as well for an address

Reply:
Added.